src/rules/mixins-no-risky-parent-selectors/README.md

```diff
@@ -0,0 +1,133 @@
+# mixins-no-risky-parent-selectors
+
+
+Parent selectors are not reliable in the following contexts:
+
+- Within a @mixin rule.
+- Nested within another style rule.
+- When they are not positioned at the beginning of a complex selector.
+- Within a @at-root rule.
+
+Using parent selectors in these scenarios may lead to unexpected and uncommon behavior.
+
+```scss
+@mixin foo {
+  .bar {
+    color: blue;
+    .baz & {      
+/*       ↑
+    * Disallow this */
+      color: red;
+    }
+  }
+}
+```
+
+A more deterministic approach would be:
```

Comment:
Nit: the behavior is always deterministic, the issue is just that it might be unexpected.